zdm/iteration.py

```diff
@@ -28,35 +29,49 @@ def get_likelihood(pset,grid,survey,norm=True,psnr=False):
 		ng=1
 		ns=1
 	if ng==1:
-		update_grid(grid,pset)
+		update_grid(grid,pset,survey)
 		if survey.nD==1:
-			loglik=calc_likelihoods_1D(grid,survey,pset,norm=norm,psnr=psnr)
+			#loglik=calc_likelihoods_1D(grid,survey,pset,norm=norm,psnr=psnr)
+			llsum,lllist,expected=calc_likelihoods_1D(grid,survey,pset,norm=norm,psnr=True,dolist=1)
 		elif survey.nD==2:
-			loglik=calc_likelihoods_2D(grid,survey,pset,norm=norm,psnr=psnr)
-		
+			#loglik=calc_likelihoods_2D(grid,survey,pset,norm=norm,psnr=psnr)
```

Comment:
remove -- same for all the other old lines of code

Reply:
Okay, will do.